doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -586,11 +586,11 @@ Requesting information
       constructed. Each hole is an existential variable, which appears as a
       question mark followed by an identifier.
 
-      Experimental: Specifying “Diffs” highlights the difference between the
+      Specifying “Diffs” highlights the difference between the
       current and previous proof step.  By default, the command shows the
       output once with additions highlighted.  Including “removed” shows
       the output twice: once showing removals and once showing additions.
-      It does not examine the :opt:`Diffs` option.  See :ref:`showing_diffs`.
+      It does not examine the :opt:`Diffs` option.  See :ref:`showing_proof_diffs`.
```

Comment:
This does not seem to work for me. If the diff mode is off (in the view menu), the key does not show the diff. See also below.

Reply:
That was my intention--to not show diffs if diff mode is off.  Perhaps I should change that.

Reply:
Note this is the description of the `Show Proof` command, not CoqIDE.  I wanted to have `Show Proof` use the `Diffs` option value automatically, but @gares insisted that I not do that.